### DISCUSSION TITLE
Ruby on Rails

### DISCUSSION THREAD

**Discussion by juliacollins** [nsidc/tech-radar#25] [Languages and Frameworks]
Ruby-based MVC framework.

**Comment by scott-lewis-nsidc** [comment]
I would agree that perhaps using the FULL "Ruby on Rails" stack may be "not recommended", but the ActiveRecord stuff is a component of Rails, and I think we should continue using that even if we switch to using Sinatra for the API/front-end stuff.  There may be other Rails components that we can leverage as well (including maybe some we aren't currently using).

**Reply by juliacollins** [reply]
@scott-lewis-nsidc I created a separate entry for ActiveRecord.
